### ISSUE TITLE
Podman driver: Invalid character(s) found in patch number "2\n"

### ISSUE THREAD

**Issue by mastertinner** [kubernetes/minikube#8746]
**Steps to reproduce the issue:** 

```shell
$ sudo minikube start --driver podman
😄  minikube v1.11.0 on Arch rolling
✨  Using the podman (experimental) driver based on user configuration

❗  'podman' driver reported an issue: Invalid character(s) found in patch number "2\n"
💡  Suggestion: Cant verify minimum required version for podman . See podman website for installation guide.
📘  Documentation: https://podman.io/getting-started/installation.html

💣  Failed to validate 'podman' driver
```

**Full output of failed command:** 

```shell
$ sudo minikube start --driver podman --alsologtostderr
I0717 12:49:14.160626  246560 start.go:98] hostinfo: {"hostname":"arch","uptime":213756,"bootTime":1594769198,"procs":273,"os":"linux","platform":"arch","platformFamily":"arch","platformVersion":"rolling","kernelVersion":"5.7.8-arch1-1","virtualizationSystem":"kvm","virtualizationRole":"host","hostid":"4c4c4544-004a-4310-8057-b6c04f4c3133"}
I0717 12:49:14.161205  246560 start.go:108] virtualization: kvm host
😄  minikube v1.11.0 on Arch rolling
I0717 12:49:14.161363  246560 driver.go:253] Setting default libvirt URI to qemu:///system
I0717 12:49:14.161406  246560 notify.go:125] Checking for updates...
I0717 12:49:14.247038  246560 podman.go:99] podman version: 2.0.2
✨  Using the podman (experimental) driver based on user configuration
I0717 12:49:14.247153  246560 start.go:214] selected driver: podman
I0717 12:49:14.247160  246560 start.go:611] validating driver "podman" against <nil>
I0717 12:49:14.247167  246560 start.go:617] status for podman: {Installed:true Healthy:false Error:Invalid character(s) found in patch number "2\n" Fix:Cant verify minimum required version for podman . See podman website for installation guide. Doc:https://podman.io/getting-started/installation.html}

❗  'podman' driver reported an issue: Invalid character(s) found in patch number "2\n"
💡  Suggestion: Cant verify minimum required version for podman . See podman website for installation guide.
📘  Documentation: https://podman.io/getting-started/installation.html

💣  Failed to validate 'podman' driver
```

System:
Arch Linux
Podman `2.0.2` installed via `sudo pacman -Syu podman`
Minikube `v1.11.0` installed via `sudo pacman -Syu minikube`

**Comment by priyawadhwa** [comment]
Hey @mastertinner thanks for opening this issue. Based on this log:

```
'podman' driver reported an issue: Invalid character(s) found in patch number "2\n"
```

I think we probably need to strip the version of the newline "\n" before attempting to parse it. This looks like a small bug in our code here:

https://github.com/kubernetes/minikube/blob/efa7f615d76f0babbd2c7538ffca3f58e42ce433/pkg/minikube/registry/drvs/podman/podman.go#L97

we would also want to trim newlines.

If anyone is interested in fixing this bug, I'm happy to review any PRs.

**Comment by mastertinner** [comment]
Thanks for the quick reply, @priyawadhwa! It seems like `strings.TrimSpace` actually trims newlines. Apparently though, this line was only added with https://github.com/kubernetes/minikube/commit/9d98e858a3 so I'm expecting it to be fixed with Minikube `1.12`.

Sorry for opening an issue for an "old" version :)

**Comment by afbjorklund** [comment]
The good news is that it was fixed in minikube 1.12.0, the bad news is that minikube doesn't work with podman 2.0

**Comment by mastertinner** [comment]
So it seems...

```shell
$ minikube start --driver podman
😄  minikube v1.12.1 on Arch rolling
✨  Using the podman (experimental) driver based on existing profile
👍  Starting control plane node minikube in cluster minikube
🔄  Restarting existing podman container for "minikube" ...
🤦  StartHost failed, but will try again: podman inspect ip minikube: sudo -n podman container inspect -f {{.NetworkSettings.IPAddress}} minikube: exit status 125
stdout:

stderr:
Error: error inspecting object: no such container minikube

🔄  Restarting existing podman container for "minikube" ...
😿  Failed to start podman container. "minikube start" may fix it: podman inspect ip minikube: sudo -n podman container inspect -f {{.NetworkSettings.IPAddress}} minikube: exit status 125
stdout:

stderr:
Error: error inspecting object: no such container minikube


💣  error provisioning host: Failed to start host: podman inspect ip minikube: sudo -n podman container inspect -f {{.NetworkSettings.IPAddress}} minikube: exit status 125
stdout:

stderr:
Error: error inspecting object: no such container minikube


😿  minikube is exiting due to an error. If the above message is not useful, open an issue:
👉  https://github.com/kubernetes/minikube/issues/new/choose
```
